README.md

```diff
@@ -1,15 +1,31 @@
-**This is being worked on again.**
-# shopdeck
-![contributors](https://img.shields.io/github/contributors/LetsShop3DS/shopdeck) ![followers](https://img.shields.io/github/followers/LetsShop3DS) ![stars](https://img.shields.io/github/stars/LetsShop3DS/shopdeck)
-
-**this project does not cover the Wii U eshop, thanks**
-
-The official source code for the replacement server!
-# Can I use this for piracy?
-No. We do NOT support piracy.
-# In what language is this written in?
-Shopdeck is written in modern Python 3.10, using 2 web frameworks: Django & Flask.
-# Is this legal?
-Yes, it is! We use no code from Nintendo servers and the base tickets we use have no data about legit titles.
-# How to setup?
-Check out the Wiki/Development Setup for that.
+# Shopdeck: Your Nintendo 3DS eShop Replacement
+
+Note That This Is Still In W.i.P ⚠️
```

Comment:
Hmmm... Could it be all lowercase, like "**Note that this is still WIP :warning:**"